doc/sphinx_source/mainDocs/tcl-commands.rst

```diff
@@ -3288,6 +3288,14 @@ The following is a list of bind types and how they work. Below each bind type is
 
   Module: core
 
+(51) INVT (stackable)
+
+  bind invt <flags> <mask> <prov>
+
+  procname <nick> <user@host> <invitee> <channel>
```

Comment:
switch "invitee" and "channel" for consistency with other binds (kick, mode) where target is last

Reply:
Done